### ISSUE TITLE
All themes loaded ungrouped from storage, despite having groups defined

### ISSUE THREAD

**Issue by beauroberts** [tokens-studio/figma-plugin#1981]
**Describe the bug**

I created a couple of themes in a theme group called "Color". When I load the JSON content attached below, the theme selector does not show the group name "Color" like I expected - instead the themes appear ungrouped.

I was able to track this down to [`src/storage/schemas/themeObjectSchema.ts`](https://github.com/tokens-studio/figma-plugin/blob/main/src/storage/schemas/themeObjectSchema.ts#L4), which does not declare `group` as a valid key. When the JSON file is validated against this schema, `group` gets filtered out.

**To Reproduce**
1. Load the JSON content attached below
2. In the theme selector, look at the hierarchy. Both the "Light" and "Dark" themes are in the ungrouped, under the heading "No group".

**Expected behavior**
The "Light" and "Dark" themes should be in a theme group called "Color".

**Screenshots or Screencasts**
<img width="583" alt="Screenshot 2023-06-22 at 17 24 43" src="https://github.com/tokens-studio/figma-plugin/assets/337712/e1fbf20b-9422-4729-b0c1-96ad11cfd77d">

**Figma file (optional)**
Send us the link to a Figma file containing a reproducible example

**JSON (optional)**

```json
{
  "global": {
    "primaryBackgroundColor": {
      "value": "#fff",
      "type": "color"
    }
  },
  "dark": {
    "primaryBackgroundColor": {
      "value": "#000",
      "type": "color"
    }
  },
  "$themes": [
    {
      "id": "light",
      "name": "Light",
      "group": "Color",
      "selectedTokenSets": {
        "global": "enabled",
        "dark": "disabled"
      }
    },
    {
      "id": "dark",
      "name": "Dark",
      "group": "Color",
      "selectedTokenSets": {
        "global": "enabled",
        "dark": "enabled"
      }
    }
  ],
  "$metadata": {
    "tokenSetOrder": [
      "global",
      "dark"
    ]
  }
}
```

**Comment by thaoula** [comment]
Hi Team,

I also commented on the pull request #1981. 

We are loading from Azure Devops and the themes are ungrouped.

Regards,
Tarek

**Comment by beauroberts** [comment]
Hi @thaoula! If you are storing your tokens across multiple files, I think it's definitely related. I only fixed the issue in [`src/storage/schemas/singleFileSchema.ts`](https://github.com/tokens-studio/figma-plugin/blob/main/src/storage/schemas/singleFileSchema.ts#L9), but it looks like [`src/storage/schemas/multiFileSchema.ts`](https://github.com/tokens-studio/figma-plugin/blob/main/src/storage/schemas/multiFileSchema.ts#L8) suffers from the same problem – the `group` key is gettin filtered out by the schema validator.

@six7 I'm happy to submit another PR for this. Should I link it to this issue or would you prefer that I create another?

**Comment by esthercheran** [comment]
Closing this issue as it gets solved by #1970
